heft-plugins/heft-jest-plugin/src/JestPlugin.ts

```diff
@@ -47,11 +54,17 @@ export interface IHeftJestConfiguration extends Config.InitialOptions {}
  * @internal
  */
 export class JestPlugin implements IHeftPlugin<IJestPluginOptions> {
+  private static _ownPackageFolder: string = path.resolve(__dirname, '..');
```

Comment:
Do you have a test that ensures that this remains valid, in case files get moved around?

Reply:
Curious, never made one for this purpose... do we have a standard way of doing this?

Reply:
I don't think so. Might be easiest to put this in a `constants.ts` file and have a test that uses `PackageJsonLookup` to verify that the path it finds is correct.